flatdata-cpp/test/ArrayViewTest.cpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2018 HERE Europe B.V.
+ * See the LICENSE file in the root of this project for license details.
+ */
+
+#include "test_structures.hpp"
+
+#include <flatdata/flatdata.h>
+#include <gtest/gtest.h>
+
+using namespace flatdata;
+using namespace test_structures;
+
+TEST( VectorTest, slicing )
+{
+    Vector< AStruct > data( 10 );
+    for ( size_t i = 0; i < 10; i++ )
+    {
+        data[ i ].value = i;
+    }
+
+    ArrayView< AStruct > view = data;
+    ASSERT_TRUE( view.size( ) == 10 );
```

Comment:
Don't you think we should also check the values?

Reply:
Done